README.rst

```diff
@@ -245,23 +272,38 @@ __ https://docs.scale.com/reference#project-update-parameters
 
     data = client.update_project(
         project_name='test_project',
-        pathc = false,
+        patch = false,
         instruction='update: Please label all the stuff',
     )
 
 Error handling
 ______________
 
 If something went wrong while making API calls, then exceptions will be raised automatically
-as a `scaleapi.ScaleException` or `scaleapi.ScaleInvalidRequest` runtime error. For example:
+as a `ScaleException` parent type and child exceptions:
+
+- ``ScaleInvalidRequest``: 400 - Bad Request -- The request was unacceptable, often due to missing a required parameter.
+- ``ScaleUnauthorized``: 401 - Unauthorized -- No valid API key provided.
+- ``ScaleNotEnabled``: 402 - Not enabled -- Please contact sales@scaleapi.com before creating this type of task.
+- ``ScaleResourceNotFound``: 404 - Not Found -- The requested resource doesn't exist.
+- ``ScaleDuplicateTask``: 409 - Conflict -- The provided idempotency key or unique_id is already in use for a different request.
+- ``ScaleTooManyRequests``: 429 - Too Many Requests -- Too many requests hit the API too quickly.
+- ``ScaleInternalError``: 500 - Internal Server Error -- We had a problem with our server. Try again later
+- ``ScaleTimeoutError``: 504 - Server Timeout Error -- Try again later.
+
+Check out `Scale's API documentation <https://docs.scale.com/reference#errors>`_ for more details.
+
+For example:
 
 .. code-block:: python
 
-    try
-        client.create_categorization_task('Some parameters are missing.')
-    except scaleapi.ValidationError as e:
-        print(e.code)  # 400
-        print(e.message)  # missing param X
+    from scaleapi.exceptions import ScaleException
+
+    try:
+        client.create_task(TaskType.TextCollection, attachment='Some parameters are missing.')
+    except ScaleException as err:
+        print(err.code)  # 400
+        print(err.message)  # Parameters is invalid, reason: "attachments" is required
```

Comment:
Is this the actual error message we provide? 😅 It should be "Parameters **are** invalid" or "The `parameters` field is invalid" I think. If this is exactly what we do return though, it's probably fine to leave as is.

Reply:
Unfortunately, this is the actual message. I'll send a PR to fix it to `Parameter is` :)